src/06-hello-world/.cargo/config

```diff
@@ -1,8 +1,8 @@
 [target.thumbv7em-none-eabihf]
```

Comment:
Can't we get rid of these files?

Reply:
Yes, I'm going to do that as update each chapter. I didn't remove them in this change because I want to update the .md files and "test" each chapter.

Reply:
Ah, I see. Let's merge this then.